src/main/java/pages/SmartphonePage.java

```diff
@@ -1,14 +1,14 @@
 package pages;
 
+import elements.CustomTextElement;
 import org.apache.log4j.Logger;
-import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
 public class SmartphonePage extends BasePage {
     private static final Logger logger = Logger.getLogger("Smartphone page logger");
 
     @FindBy(xpath = "//*[contains(@class,'schema-header__title') and text()='Мобильные телефоны']")
-    private WebElement header;
+    private CustomTextElement header;
```

Comment:
Please update element name. Element's names should contain elemeent type (good practice), ex. headerText and etc